app/src/main/java/org/hwyl/sexytopo/control/util/SurveyUpdater.java

```diff
@@ -222,12 +222,16 @@ public static synchronized void editLeg(
 
 
     public static void renameStation(Survey survey, Station station, String name) {
+        String previousName = station.getName();
+        Log.d("Renaming station from: " + Log.formatDebugString(previousName) + " to: " + Log.formatDebugString(name));
```

Comment:
This logging might be _too_ verbose now. I thought it was useful to include the input value in addition to what it was finally set to. This could be combined into a single log statement.